sig-architecture/api-review-process.md

```diff
@@ -1,10 +1,12 @@
 # Kubernetes API Review Process
 
-[API Approvers](mailto:kubernetes-api-reviewers@googlegroups.com): [Clayton Coleman](https://github.com/smarterclayton), [Jordan Liggitt](https://github.com/liggitt), [Tim Hockin](https://github.com/thockin), [Brian Grant](https://github.com/bgrant0607), [Eric Tune](https://github.com/erictune)
+[API Approvers](mailto:kubernetes-api-reviewers@googlegroups.com): [Clayton Coleman](https://github.com/smarterclayton), [Jordan Liggitt](https://github.com/liggitt), [Tim Hockin](https://github.com/thockin), [Brian Grant](https://github.com/bgrant0607)
+
+Emeritus Approvers: [Eric Tune](https://github.com/erictune)
```

Comment:
suggest dropping all the individual names and referencing:
* the mailing list for discussions
* the `api-approvers` and `<sig>-api-reviewers` aliases in [OWNERS_ALIASES](https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES)

Reply:
+1 I'll make that change.